File: fib/fib.go
package main

import (
	"fmt"
	"time"
)

func fib_rec(n int) int {
	if n == 1 || n == 2 {
		return 1
	}
	return fib_rec(n-1) + fib_rec(n-2)
}
func main() {
	startTime := time.Now()
	fib_rec(30)
	endTime := time.Since(startTime)
	fmt.Println(endTime)
}

File: fib/fib.php
<?php
function fib($n){
    if($n == 1 || $n == 2) return 1;
    return fib($n - 1) + fib($n - 2);
}
$start_time = hrtime(true);
fib(30);
$endtime = hrtime(true);
echo ($endtime - $start_time)/1e+6," ms\n";